src/main/java/Duke.java

```diff
@@ -1,10 +1,48 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String DIVIDER = "-----------------------------------------";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        TaskManager taskManager = new TaskManager();
+
+        boolean isExit = false;
+        Scanner in = new Scanner(System.in);
+        do {
+            String userInput = in.nextLine();
+            String[] words = userInput.split(" ");
```

Comment:
It is better use another way to extract the command, since you create a array of String, but only use the first element.
```suggestion
            String word = userInput.substring(0, userInput.indexOf(" "););
```